reliability-scanner/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.15.7 AS build
+
+WORKDIR /go/src/github.com/vmware-tanzu/sonobuoy-plugins/reliability-scanner
+
+COPY go.mod go.sum ./
+COPY cmd cmd
+COPY internal internal
+COPY api api
+
+ENV CGO_ENABLED=0
+RUN go mod download
+RUN go build -o /go/bin/ ./cmd/reliability-scanner/
+
+FROM scratch AS final
```

Comment:
Is there a specific reason to use `scratch` over [`distroless`](https://github.com/GoogleContainerTools/Distroless)? We generally recommend `distroless` because it preserves TZ data, etc.

Reply:
No problems @wilsonehusin I have updated and tested using `distroless`.